kubernetes/roles/deploy-kubevirt/tasks/main.yml

```diff
@@ -1,14 +1,97 @@
-- name: "List all manifests"
-  find:
-    paths: "{{ go_path }}/{{ kubevirt_dir }}/manifests"
-    patterns: "*.yaml"
-  register: all_manifests_files
+---
```

Comment:
Can you remove commented section, it was actually only for development environment